CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+## 1.1.9
+ * Fix to allow `hs filemanager fetch` to fetch all files
+ * `hs hubdb` now supports referencing tables by name or ID
```

Comment:
I'd add a bullet about releasing `hs hubdb` into BETA and publicly documenting it.